.travis.yml

```diff
@@ -48,4 +48,5 @@ jobs:
 branches:
   only:
   - master
+  - estuary_umbrella
```

Comment:
This doesn't seem consistent with how we've done the other apps. Are we using this just temporarily and then planning on removing him once we're no longer branching from a branch?

Reply:
yes